### PR TITLE
Added method to get context from model or store and another to expand the url using a context

### PR DESCRIPTION
**Pull request by crisely09**
Related to [this issue](https://bbpteam.epfl.ch/project/issues/browse/BBPP134-536).

- Added methods to get a context from a store and from a model
- Added method to extend an URI using a given context

### CODE REVIEW

**Review by MFSY — CHANGES_REQUESTED** [review]

File: kgforge/core/forge.py [hunk 1]
```diff
@@ -215,12 +218,12 @@ def __init__(self, configuration: Union[str, Dict], **kwargs) -> None:
         self._model: Model = model(**model_config)
 
         # Store.
-
         store_config.update(model_context=self._model.context())
         store_name = store_config.pop("name")
         store = import_class(store_name, "stores")
         self._store: Store = store(**store_config)
         store_config.update(name=store_name)
+        self._store.mapper = DictionaryMapper(self)
```
[the remark this answers is not in the document]
Comment (crisely09) [reply kgforge/core/forge.py]: this change was made

File: kgforge/core/forge.py [hunk 2]
```diff
@@ -447,14 +456,50 @@ def resolve(
...
+    def expand_uri(self, uri: str, context: Context = None, 
+                   is_file: bool = True, encoding: str = None):
+        """
+        Construct an URI a given an id using the vocabulary given in the Context object.
+        
+        :param uri: the uri to transform
+        :param context: a Context object that should be used to create the URI
+        :param encode: parameter to use to encode or not the uri, default is `utf-8`
+        """
+        print("in eexpada URI")
```
Comment (crisely09) [inline kgforge/core/forge.py:511]: This should be removed.

Comment (MFSY) [reply kgforge/core/forge.py]: done

File: kgforge/specializations/stores/bluebrain_nexus.py [hunk 3]
```diff
@@ -135,9 +135,9 @@ def mapping(self) -> Optional[Callable]:
         return DictionaryMapping
 
     @property
-    def mapper(self) -> Optional[Callable]:
-        return DictionaryMapper
-
+    def mapper(self) -> Optional[DictionaryMapper]:
+        return  DictionaryMapper
```
[the remark this answers is not in the document]
Comment (MFSY) [reply kgforge/specializations/stores/bluebrain_nexus.py]: done

**Review by MFSY — APPROVED** [review]

**Comment by crisely09** [comment]
To me, it is ready to merge.

**Pull request merged**